packages/extras/src/lib/components/HTML/HTML.svelte

```diff
@@ -124,7 +125,7 @@
     }
   })
 
-  useTask(() => {
+  export const render = () => {
```

Comment:
hold up, why `export`? ... wait nvm, found the svelte docs for it: https://svelte.dev/docs/svelte/legacy-export-let#Component-exports. makes sense.

Reply:
oh yeah just a component export. it's interesting `export let` means two different things in svelte 4 and 5.